packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagebubble/_mixins.scss

```diff
@@ -0,0 +1,45 @@
+@mixin chat-messagebubble(
+  $padding,
+  $border-radius,
+  $background-color-secondary,
+  $background-color-primary,
+  $line-height,
+  $bubble-container-gap
+) {
+  .dx-chat-message-bubble-container {
```

Comment:
This is a mistake that width of all messages inside group is 100%

<img width="471" alt="image" src="https://github.com/user-attachments/assets/34b48f3f-3505-482e-b944-9f83d79e918a">

Reply:
[b5bbab3e3d2663f50b15f1c50726ddb07f6b6094]